Models/Product.cs

```diff
@@ -27,11 +27,15 @@ public class Product
     public decimal Price { get; set; }
 
     [Required]
+    public int ProductSubTypeId { get; set; }
+
```

Comment:
You'll need `public ProductSubType ProductSubType { get; set; }` here to set up the foreign key dependency with EF